packages/nimble-components/package.json

```diff
@@ -139,6 +139,7 @@
     "eslint-plugin-storybook": "^0.8.0",
     "html-webpack-plugin": "^5.3.1",
     "jasmine-core": "^5.1.2",
+    "js-beautify": "^1.15.1",
     "karma": "^6.3.0",
```

Comment:
Can you remove the explicit "prettier": "^3.2.5" dependency below? (we can just rely on whatever prettier-eslint brings in)
Make sure to re-run npm install after for the updated lockfile